source/scid/linalg.d

```diff
@@ -418,10 +418,22 @@ body
 
 template DetType(MatrixT)
 {
-    static if (scid.core.traits.isComplex!(BaseElementType!MatrixT))
```

Comment:
However, the unittest on line 479 is failing due to the reduced width of the return type (the determinant becomes `double.infinity`). You don't see it because the unittest system uses a custom `check()` function instead of `assert`, which means that `dub test`, and by extension Travis, doesn't catch it. (I am going to fix that by replacing all `check()` statements with `assert` immediately.)